### ISSUE TITLE
[Conflict] cosmetics will blocked by anti-cheat mod

### ISSUE THREAD

**Issue by CoolLKKPS** [Charlese2/HostFixes#16]
https://github.com/notnotnotswipez/MoreCompany/issues/73

**Comment by CoolLKKPS** [comment]
@Charlese2

**Comment by Charlese2** [comment]
I will start looking into it tomorrow.

**Comment by 1A3Dev** [comment]
The morecompany cosmetic syncing was moved from AddPlayerChatMessageServerRpc to AddTextMessageServerRpc so it should be a fairly simple fix https://github.com/notnotnotswipez/MoreCompany/blob/master/MoreCompany/ChatPatches.cs#L61

**Comment by CoolLKKPS** [comment]
@Charlese2 and also another issue
cheater can still comeback after kicked out, i tested that with normal player, and yes with more company

**Comment by CoolLKKPS** [comment]
additional: only happen when online, and maybe when other player don't have hostfixes mod
